bindata/assets/alerts/cpu-utilization.yaml

```diff
@@ -21,8 +21,16 @@ spec:
               To fix this, increase the CPU and memory on your control plane nodes.
           expr: |
             sum(
-              100 - (avg by (instance) (rate(node_cpu_seconds_total{mode="idle"}[1m])) * 100)
-              AND on (instance) label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
+              100 - (
+                avg by (instance) (
+                  sum without (mode) (
+                    rate(
+                      node_cpu_seconds_total{mode=~"idle|iowait|steal"}[1m]
```

Comment:
Is there separate alert coverage for "high overall control-plane iowait, maybe get faster disks or scale down your consumption"?

Reply:
I don't think we have one, but it would definitely make sense especially considering the history with I/O on Azure

Reply:
#1553 is poking around in this space now :)

Reply:
yes I started a discussion with the etcd team in https://redhat-internal.slack.com/archives/C027U68LP/p1695210456306999, but we still need to figure out if alerting on iowait is the right thing to do